carbon-design-system/carbon
# feat: add actionable notification

Closes #8703

Adds `ActionableNotification` as a new component and modify `Inline`/`Toast` notification components to no longer accept actions. The aim here is to promote a more clear onramp for best accessibility practices with notifications.

#### Changelog

**New**

- actionable-notification: add new component, styles, storybook docs

**Changed**

- inline-notification: no longer accept actions or interactive children
- toast-notification: no longer accept actions or interactive children

#### Testing / Reviewing

- View the carbon-next storybook, all 3 notification stories
- Ensure various controls/knobs work and the component renders as expected
- Particuarly look at the ActionableNotification component and validate all variants render appropriately, _across the 4 different themes too_
- View the legacy/normal storybook and make sure the existing Toast/Inline are not impacted with changes from `next`
